build.ps1

```diff
@@ -0,0 +1,15 @@
+echo "Stopping all running docker containers"
```

Comment:
Acho melhor deixar esses scripts em uma pasta separada. Uma pasta `scripts` talvez

Reply:
Consider it done!